release/major-cockpit-release

```diff
@@ -17,8 +17,8 @@ job release-srpm
 
 # Do fedora builds for the tag, using tarball
 job release-koji -k master
-job release-koji f25
-job release-koji -k f26
+job release-koji -k f25
```

Comment:
I don't think we should commit to dist-git here automatically at all. Because the next time (if) we do, we would want to then backport a patch, and a bunch of other useless dist-git commits would be in the way. I think we should just remove this line.